README.md

```diff
@@ -25,7 +25,7 @@ A list of release versions is available at https://github.com/MeltanoLabs/tap-gi
 This tap accepts the following configuration options:
 
 - Required: One and only one of the following modes:
-  1. `repositories`: an array of strings containing the github repos to be included
+  1. `repositories`: an array of strings containing the github repos to be included. The repo URL prefix should be excluded (e.g. MeltanoLabs/tap-github).
```

Comment:
```suggestion
  1. `repositories`: an array of strings specifying the GitHub repositories to be included. Each element of the array should be of the form `<org>/<repository>`, e.g. `MeltanoLabs/tap-github`.
```

Reply:
I think it's more clear if we say specifically what it ought to be, rather than what to exclude. I wasn't sure what you meant by "repo URL prefix" at first.

Reply:
@WillDaSilva yep 100% thanks for the suggestion!